lib/engine/game/base.rb

```diff
@@ -1034,7 +1034,7 @@ def place_home_token(corporation)
         city.place_token(corporation, token)
       end
 
-      def tile_cost(tile, hex, entity)
+      def tile_cost(tile, hex, entity, new_tile: nil) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
was new_tile really necessary? can't you just do hex.tile?

Reply:
Good point. Didn't realize `hex.lay` had already been called by now.